### ISSUE TITLE
SuperStructure.

### ISSUE THREAD

**Issue by lostpreviousaccoun** [studio-minus/ppg-bugs-and-requests#1363]
**Feature description**
A large, Above ground structure set either in a snowy* wasteland or directly on the exterior of the facility, Like The Wall in rainworld. 

Features many of the same rooms as SubStructure, But above ground.
(and snowy.) Would probably require local temprature if the map is based in the examples. But that is sacrificable.

**Feature purpose**
Its a map to play around on, Like every other map.
**Additional context**
Below are two examples. DO NOT USE THE LAYOUT SEEN! It is a placeholder since I'm shit at map design. I intended to make a real layout but I messed it up.
![very bad example](https://user-images.githubusercontent.com/105872521/169838122-be9fee1c-5024-4e99-8069-4cd9b7f5e70f.PNG)
![other dedign](https://user-images.githubusercontent.com/105872521/169839378-30803b24-6309-4b47-9539-bca3a4b369b6.PNG)

**Comment by BBQGiraffe** [comment]
this would be cool if the structure was on the face of a mountain, the first outdoor map that isn't from a mod

**Comment by lostpreviousaccoun** [comment]
Snow technically is outdoors. Its just a terraformed cube in a snowy area, The blue is clearly sky.

**Comment by WWWOWWWPL** [comment]
holy damn this would be cool

**Comment by Aeternaliter** [comment]
Yeah, would be cool if it had some kind of bottomless pit like the void

**Comment by lostpreviousaccoun** [comment]
> Yeah, would be cool if it had some kind of bottomless pit like the void

I was actually going to add that to the suggestion, But I restructured the text description to be more readable and just forgot lol.

**Comment by TheBeeThat** [comment]
> Snow technically is outdoors. Its just a terraformed cube in a snowy area, The blue is clearly sky.

the snow map isnt outside, there is fans for the snow
and i dont think the blue is really the sky
![image](https://user-images.githubusercontent.com/96020070/172061595-ed800b91-dc42-41d7-ae5e-c03c73ce29f8.png)
